pkg/section/module.go

```diff
@@ -0,0 +1,75 @@
+package section
+
+import (
+	"fmt"
+	"strings"
+
+	"golang.org/x/tools/go/packages"
+
+	"github.com/daixiang0/gci/pkg/parse"
+	"github.com/daixiang0/gci/pkg/specificity"
+)
+
+const ModuleType = "module"
+
+type Module struct {
+	ModulePaths []string
+}
+
+func (m *Module) MatchSpecificity(spec *parse.GciImports) specificity.MatchSpecificity {
+	for _, modPath := range m.ModulePaths {
+		if strings.HasPrefix(spec.Path, modPath) {
+			return specificity.Module{}
+		}
+	}
+
+	return specificity.MisMatch{}
+}
+
+func (m *Module) String() string {
+	return ModuleType
+}
+
+func (m *Module) Type() string {
+	return ModuleType
+}
+
+// Configure configures the module section by finding the module
+// for the current path
+func (m *Module) Configure() error {
+	modPaths, err := findModules()
+	if err != nil {
+		return err
+	}
+	m.ModulePaths = modPaths
+	return nil
+}
+
+func findModules() ([]string, error) {
+	packages, err := packages.Load(
+		// find the package in the current dir and load its module
```

Comment:
Please update README and highlight it -- we should run GCI at root dir or dir including go.mod.

Reply:
> Please update README and highlight it -- we should run GCI at root dir or dir including go.mod when enable this feature.

https://github.com/daixiang0/gci/pull/179/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5